bundle/manifests/self-node-remediation.clusterserviceversion.yaml

```diff
@@ -40,7 +40,7 @@ metadata:
     createdAt: "2022-11-06 08:57:52"
     description: Self Node Remediation Operator for remediate itself in case of a
       failure.
-    olm.skipRange: '>=0.1.0 <0.5.0'
+    olm.skipRange: '>=0.4.0 <0.5.0'
```

Comment:
Since 0.4.0 was the first release, this change does not make a difference in the end. But it also doesn't hurt.

Reply:
Agreed, we've done it awhile back (this PR is just a leftover because I didn't generate csv properly) just for good measure.